### PR TITLE
fix the url of readme in git paper

### PR DESCRIPTION
**Pull request by shikanon**
refer #1221 
fix the bug of invalid link in blog readme

### CODE REVIEW

File: docs/blog/zh/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # 博客
 
 
-- [KubeVela 正式开源：一个高可扩展的云原生应用平台与核心引擎](/blog/zh/kubevela-the-extensible-app-platform-based-on-open-application-model-and-kubernetes.md)
-- [如何在 20 分钟内给你的 K8s PaaS 上线一个新功能](/blog/zh/extend-kubevela-by-cuelang-in-20-mins.md)
\ No newline at end of file
+- [KubeVela 正式开源：一个高可扩展的云原生应用平台与核心引擎](/docs/blog/zh/kubevela-the-extensible-app-platform-based-on-open-application-model-and-kubernetes.md)
+- [如何在 20 分钟内给你的 K8s PaaS 上线一个新功能](/docs/blog/zh/extend-kubevela-by-cuelang-in-20-mins.md)
```
Comment (wonderflow) [inline docs/blog/zh/README.md:5]: If changed, will it work on the website? https://kubevela.io/#/blog/zh/

Comment (shikanon) [reply docs/blog/zh/README.md:5]: The website is generate by the docs? Where the code, I fix it also.

Comment (wonderflow) [reply docs/blog/zh/README.md:5]: Use the same doc, so we can read it normally from website but error link in raw github link.

Comment (shikanon) [reply docs/blog/zh/README.md:5]: I think it needs to be unified by repairing the official website configuration. For example, set website add prefix `/docs` to support. Or move blog page to the new repo to be independent.

Comment (wonderflow) [reply docs/blog/zh/README.md:5]: yes, we could discuss more in Dingtalk with @sunny0826

**Comment by sunny0826** [comment]
NO...This link is used to generate the blog module on the official kubevela website.
Do not merge.
image

**Review by sunny0826 — CHANGES_REQUESTED** [review]
This will affect the normal use of the official website.

**Comment by shikanon** [comment]
> This will affect the normal use of the official website.

would you change the official website configuration? This invalid URL is annoying. I have two solutions:
1. To independent official documentation blog for other repository
2. To add route for blog website make readme link and official website compatible,  change configuration like this:
```json
location /docs/blog/ {
    proxy_pass http://127.0.0.1/blog/;
}
```

**Comment by sunny0826** [comment]
The official website is being upgraded, and the upgraded website will not have this problem anymore.

**Comment by wonderflow** [comment]
closing, the blog will migrate to kubevela.io

**Pull request closed without merging**